nntrainer/compiler/remap_realizer.h

```diff
@@ -28,7 +28,28 @@ namespace nntrainer {
  */
 class RemapRealizer final : public GraphRealizer {
 public:
-  RemapRealizer(std::function<void(std::string &)> remap_function);
+  /**
+   * @brief Construct a new Remap Realizer object (connection mode)
+   *
+   * @param remap_connection_function remap connection function, with this
+   * constructor, only connections will be remapped eg) if you are inserting a
+   * new layer node in between, only connections need remapping
+   */
+  RemapRealizer(
+    std::function<void(std::string & /**< identifier */,
```

Comment:
remap_function was remapping the given identifier and modify the string.
Just to clarify, remap_connection_function will modify the index and the string both right?

Reply:
yes :)